bears/xml/XMLBear.py

```diff
@@ -24,10 +26,16 @@ class XMLBear(LocalBear, Lint):
     gives_corrected = True
 
     def process_output(self, output, filename, file):
-        if self.stdout_output:  # only yield Result if stdout is not empty
-            return self._process_corrected(self.stdout_output, filename, file)
-        if self.stderr_output:  # pragma: no cover
+        if self.stderr_output:
+            self.use_stderr = True
+            if self.stdout_output:  # only yield Result if stdout is not empty
+                return collections.ChainMap(
+                           self._process_issues(self.stderr_output, filename),
+                           self._process_corrected(self.stdout_output,
+                                                   filename, file))
             return self._process_issues(self.stderr_output, filename)
+        elif self.stdout_output:  # only yield Result if stdout is not empty
```

Comment:
@AbdealiJK It is specific to that ``elif`` clause. Won't it yield result only when there is some correction to be made ?